docs/DeveloperGuide.md

```diff
@@ -300,6 +300,7 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 | `* *`    | Teaching Assistant                                        | sort students by their progress (based on the number of tasks completed)               | see how the class is performing                                                               |
 | `* *`    | Teaching Assistant                                        | edit details of a task given to a student                                              | update the details of an assignment in case of future changes/extensions                      |
 | `* *`    | Teaching Assistant                                        | assign tasks to multiple students                                                      | allocate tasks to students without having to individually assign them                         |
+| `* *`    | Teaching Assistant                                        | delete multiple tasks assigned to students                                             | remove any tasks that has incorrect information without having to individually delete them    |
```

Comment:
```suggestion
| `* *`    | Teaching Assistant                                        | delete a task assigned to multiple students                                             | remove any task that has incorrect information without having to individually delete from each student |
```

Reply:
Edited this user story and changed the line spacing